docs/defining-tests/language.test.yaml

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 # The most typical use pattern is the following:
+type: test/samples
```

Comment:
Why is there a type but no schema version?

Can we introduce a `config_schema_version` version if/when we're introducing a `type`?

Preferably a `type` and `config_schema_version` pair for consistency. 100% of current sample tester users author these tests alongside gapic sample .yaml files.

Reply:
Filed #110 and will do this in a separate PR right after this one.